packages/website/src/containers/content/pages/OfficerOverviewPage.tsx

```diff
@@ -26,9 +68,56 @@ export const OfficerOverviewPage: React.FC<OfficerOverviewPageProps> = ({ page }
         ))}
       </Sectionbar>
 
-
       <div className="LokiContainer">
-        <h1>Officer page</h1>
+        <div className="Layout Layout--sidebar-left" css={topOverviewStyles}>
+          <div>
+            <OneImage
+            aspectRatio={AspectRatio.r1by1}
+            src={page.officerImage.resource}
+            alt=""
```

Comment:
Always set an image alt, we can either use the title of the image from wagtail, or set within the template. Such as "Headshot of President, Frida" etc